Basic-Car-Maintenance/Shared/MainView/Views/AlertView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AlertView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Omar Hegazy on 11/10/2023.
+//
+
+import SwiftUI
+
+struct AlertView: View {
+    @Environment(\.dismiss) var dismiss
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                ScrollView {
+                    VStack(alignment: .center) {
+                        Text("🤠")
+                            .font(.system(size: 100))
+                        
+                        Text("This is the title")
+                            .font(.title)
+                            .lineLimit(2)
+                            .bold()
+                        
+                        Text("Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.")
+                            .multilineTextAlignment(.center)
+                    }
+                }
+                .scrollIndicators(.hidden)
+                .padding()
```

Comment:
Change this `.padding()` to `.padding(.horizontal, 24)`

Because otherwise there's this awkward line here at the top

<img width="359" alt="Screenshot 2023-10-11 at 10 08 45 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/94a2de9b-93b3-40ec-ba89-05fea08c1ef1">